src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/wolox_training_db
+spring.datasource.username=postgres
+spring.datasource.password=lala123
```

Comment:
@JoelIvanStella - I understand that in this instance it is not necessary, but normally these keys are usually set as environment variables so they are not exposed in a repository. Maybe you knew this, but if not, advice never hurts :smile:

Reply:
Yes, thank Mati!